packages/sampler/stories/qa/VirtualList.js

```diff
@@ -123,14 +139,44 @@ const InPanels = ({className, title, ...rest}) => {
 };
 
 storiesOf('VirtualList', module)
+	.add(
+		'horizontal scroll in Scroller',
```

Comment:
I noticed that this sample exposes a really bad behavior in horizontal VL.  Repeated pressing of the horizontal scroll buttons causes the list to scroll back before going forward.  I have a video showing this if needed.  Interestingly, that bounceback gets reset if you perform other actions before scrolling with the paging controls.

Reply:
As we discussed, this issue was related with ENYO-6110. So let's investigate it more in it.